.readthedocs.yml

```diff
@@ -13,6 +13,6 @@ python:
         - all
 
 conda:
-  environment: environment.yml
+  environment: docs/rtd-environment.yml/environment.yml
```

Comment:
Should this be ` environment: docs/rtd-environment.yml`?

Reply:
I figured out how to add a commit to your PR easily, so don't worry about changing this, I got it!